src/main/fc/fc_tasks.c

```diff
@@ -139,6 +143,14 @@ static void taskUpdateRxMain(timeUs_t currentTimeUs)
 
     isRXDataNew = true;
 
+#ifdef USB_CDC_HID
+    int8_t report[8];
+    for (int i = 0; i < 8; i++) {
+    		report[i] = scaleRange(constrain(rcData[i], 1000, 2000), 1000, 2000, -127, 127);
+    }
+    USBD_HID_SendReport(&USB_OTG_dev, (uint8_t*)report, 8);
```

Comment:
Is this call safe before USB initialization?

Reply:
Yep that's fine, the function check if USB device is actually connected otherwise it doesn't sent the report.